frontend/src/assets/img/icons/dislike-icon.svg

```diff
@@ -1,6 +1,6 @@
 <g id="dislike 4" clip-path="url(#clip0_2_8877)">
-<path id="Vector" d="M12.2654 0H10.8872C10.3553 0 9.92255 0.425208 9.92255 0.947917V1.14617C9.08905 0.729083 7.19604 0 4.11728 0H3.23527C1.95691 0 0.872588 0.898083 0.655393 2.13525L0.0379868 5.65608C-0.093212 6.40738 0.117919 7.17113 0.616254 7.75288C1.11459 8.33354 1.84445 8.66667 2.61786 8.66667H5.11836C4.96235 9.07833 4.80635 9.6785 4.80635 10.4271C4.80635 12.4621 5.97831 13 6.59793 13C7.74233 13 7.83825 11.9513 7.83825 10.9688C7.83825 9.7695 9.20702 8.77283 9.92971 8.32975C9.96665 8.81942 10.3795 9.20833 10.8872 9.20833H12.2654C12.7973 9.20833 13.2301 8.78313 13.2301 8.26042V0.947917C13.2301 0.425208 12.7973 0 12.2654 0ZM7.01137 10.9688C7.01137 12.1875 6.80134 12.1875 6.59793 12.1875C6.20984 12.1875 5.63323 11.7184 5.63323 10.4271C5.63323 9.22675 6.11172 8.49117 6.11503 8.48575C6.19937 8.36117 6.20764 8.20083 6.13542 8.06867C6.06376 7.9365 5.92319 7.85417 5.77104 7.85417H2.61786C2.08866 7.85417 1.58977 7.62667 1.24909 7.22963C0.907868 6.83204 0.763439 6.30879 0.853294 5.79475L1.4707 2.27392C1.61899 1.42675 2.36098 0.8125 3.23527 0.8125H4.11728C7.86747 0.8125 9.74615 1.96517 9.92255 2.07946V7.39321C9.46335 7.63263 7.01137 9.00304 7.01137 10.9688ZM12.4032 8.26042C12.4032 8.33517 12.3415 8.39583 12.2654 8.39583H10.8872C10.8112 8.39583 10.7494 8.33517 10.7494 8.26042V0.947917C10.7494 0.87425 10.8123 0.8125 10.8872 0.8125H12.2654C12.3415 0.8125 12.4032 0.873167 12.4032 0.947917V8.26042Z" fill="#44996B"/>
+<path id="Vector" d="M12.2654 0H10.8872C10.3553 0 9.92255 0.425208 9.92255 0.947917V1.14617C9.08905 0.729083 7.19604 0 4.11728 0H3.23527C1.95691 0 0.872588 0.898083 0.655393 2.13525L0.0379868 5.65608C-0.093212 6.40738 0.117919 7.17113 0.616254 7.75288C1.11459 8.33354 1.84445 8.66667 2.61786 8.66667H5.11836C4.96235 9.07833 4.80635 9.6785 4.80635 10.4271C4.80635 12.4621 5.97831 13 6.59793 13C7.74233 13 7.83825 11.9513 7.83825 10.9688C7.83825 9.7695 9.20702 8.77283 9.92971 8.32975C9.96665 8.81942 10.3795 9.20833 10.8872 9.20833H12.2654C12.7973 9.20833 13.2301 8.78313 13.2301 8.26042V0.947917C13.2301 0.425208 12.7973 0 12.2654 0ZM7.01137 10.9688C7.01137 12.1875 6.80134 12.1875 6.59793 12.1875C6.20984 12.1875 5.63323 11.7184 5.63323 10.4271C5.63323 9.22675 6.11172 8.49117 6.11503 8.48575C6.19937 8.36117 6.20764 8.20083 6.13542 8.06867C6.06376 7.9365 5.92319 7.85417 5.77104 7.85417H2.61786C2.08866 7.85417 1.58977 7.62667 1.24909 7.22963C0.907868 6.83204 0.763439 6.30879 0.853294 5.79475L1.4707 2.27392C1.61899 1.42675 2.36098 0.8125 3.23527 0.8125H4.11728C7.86747 0.8125 9.74615 1.96517 9.92255 2.07946V7.39321C9.46335 7.63263 7.01137 9.00304 7.01137 10.9688ZM12.4032 8.26042C12.4032 8.33517 12.3415 8.39583 12.2654 8.39583H10.8872C10.8112 8.39583 10.7494 8.33517 10.7494 8.26042V0.947917C10.7494 0.87425 10.8123 0.8125 10.8872 0.8125H12.2654C12.3415 0.8125 12.4032 0.873167 12.4032 0.947917V8.26042Z" fill='black'/>
```

Comment:
Try to set `fill='currentColor`' instead, and then you can change the SVG's color by setting the `color` CSS property on a parent element, or SVG itself. I guess then there won't be any need to put `path` in selector here `frontend/src/pages/articles/components/article-card/styles.module.scss`

Reply:
Thank you